docs/index.rst

```diff
@@ -31,8 +31,8 @@ Architecture and Best Practices
    * - `Architecture and Engineering Confluence page <https://openedx.atlassian.net/wiki/spaces/AC/overview>`_
      - Confluence page for notes, thoughts, and project-related documents on Open edX architecture and engineering.
 
-   * - `(New) Open edX Developer's Guide <developers_guide/index.html>`_
-     - General guidelines for developing on various parts of the Open edX code base.
+   * - `Open edX Micro-frontend Developer's Guide <micro_frontends/index.html>`_
```

Comment:
```suggestion
   * - :doc:`micro_frontends/index`
```
This is a link within the book.  Using a .html URL renders it as a jump out of the book.  This form of link also gets the title from the index.rst page, reducing duplication.

Reply:
Thank you!  Learning to 🎣 a bit here.